packs/clicrud/actions/config_command.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: config_command
+runner_type: python-script
+description: Login in to a Brocade ICX/MLX and run a configuration command
+enabled: true
+entry_point: config_command.py
+parameters:
+  host:
+    type: string
+    description: Valid IPv4 address or hostname that resolves.
+    required: true
+  command:
+    type: array
+    description: List of commands separated by commas.
```

Comment:
Just a heads up - "string separated by comma" is a CLI and UI convenience thing for supplying a list. if your provide parameter value via API or similar, you need to provide an actual JSON array.